### ISSUE TITLE
[Screen Reader - Make Code - Documentation] Landmarks are not defined properly on the Documentation page.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2808]
**User Experience:**
User who depends on Keyboard with screen reader may not able to skip the content and navigate to the required section if landmarks are not defined properly.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.2)
Platform: Edge.

**Repro Steps:-**
1. Navigate to https://makecode.microbit.org/projects
2. Navigate in landmarks mode of narrator(Caps + Up/Down arrow Keys) and verify the issue.

**Actual Result:-**
Landmarks are not defined properly on the Documentation page.

**Expected Result:-**
Landmarks should be defined properly on the Documentation page.

**Note:** 
This issue exists with Firefox also.

**MAS Reference -**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Refer Attachment**
[MAS1.3.1_MakeCode_Landmarks are not properly defined.zip](https://github.com/Microsoft/pxt/files/1248965/MAS1.3.1_MakeCode_Landmarks.are.not.properly.defined.zip)

**Comment by veler** [comment]
Already fixed but not deployed yet.

**Comment by v-mesala** [comment]
Verified on URL:https://makecode.microbit.org/projects and issue is fixed now. Hence closing the bug.
